### PR TITLE
Update text in e2e test docs

### PR DESCRIPTION
**Pull request by SgtCoDFish**
Changes made while I was using the docs myself.

This arises from implementing https://github.com/jetstack/cert-manager/pull/4232 which arises from  https://github.com/jetstack/cert-manager/issues/2820.

/kind documentation

### CODE REVIEW

File: content/en/docs/contributing/e2e.md [hunk 1]
```diff
@@ -5,91 +5,97 @@ weight: 70
...
 
-> Note: you can see the status of each commit on the master branch at
-> [`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master).
-> You can join the Google group
-> [`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
-> in order to receive a notification by email whenever a commit on master
-> fails.
+Unless you've made huge changes to the cert-manager codebase - or to the end-to-end
+tests themselves - you probably don't _need_ to run the tests locally. If you do want to
+run the tests, though, this document explains how.
```
Comment (maelvls) [inline content/en/docs/contributing/e2e.md:16]: nit: I assumed by `-` you meant em-dashes as per https://practicaltypography.com/trademark-and-copyright-symbols.html? Three dashes `---` are rendered as an em-dash in Hugo:

```suggestion
Unless you've made huge changes to the cert-manager codebase---or to the end-to-end
tests themselves---you probably don't need to run the tests locally. If you do want to
run the tests, though, this document explains how.
```

Before:

image


After:

image

File: content/en/docs/contributing/e2e.md [hunk 2]
```diff
@@ -5,91 +5,97 @@ weight: 70
...
 
-It is only advised to run this locally when you made big changes to the
-codebase. This document explains how you can run the end-to-end tests yourself.
+> Note: the status of each commit on the master branch is reported on
+> [`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master).
+> Join the [`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
+> Google group to receive email notifications when tests fail.
```
Comment (maelvls) [inline content/en/docs/contributing/e2e.md:21]: nit: would it be worth using a shortcode here? (not sure if my proposition is an improvement or not aesthetically wise...)

```suggestion

{{% alert title="TestGrid alerts" color="" %}}

The status of each commit on the master branch is reported on
[`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master). Join
the
[`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
Google group to receive email notifications when tests fail.

{{% /alert %}}
```

Before:

image

After:

image

**Review by maelvls — APPROVED** [review]
Thank you for the improvements!! 👍

A few nits but feel free to unhold.

/lgtm
/hold

File: content/en/docs/contributing/e2e.md (hunk 1 above)
Comment (SgtCoDFish) [reply content/en/docs/contributing/e2e.md:16]: Did you mean to link to that page specifically? I'm not sure how it relates to dashes

I'm very indifferent to dash styling; I'd be totally content seeing and using `-` on any page or in any piece of text. I'll switch to emdashes though just to be fancy :wink:

File: content/en/docs/contributing/e2e.md (hunk 2 above)
Comment (SgtCoDFish) [reply content/en/docs/contributing/e2e.md:21]: oh yeah, that's much better. I'll use that, thanks :+1:

**Comment by maelvls** [comment]
/lgtm

Thank you!!

**Pull request merged**